vendor/bat-native-ads/src/bat/ads/internal/legacy_migration/database/database_migration_unittest.cc

```diff
@@ -13,6 +13,9 @@
 #include "bat/ads/internal/creatives/creative_ad_info.h"
 #include "bat/ads/internal/creatives/creative_ad_unittest_util.h"
 #include "bat/ads/internal/legacy_migration/database/database_constants.h"
+#include "bat/ads/internal/processors/contextual/text_embedding/text_embedding_html_event_info.h"
```

Comment:
All changes to this file should be reverted, as we should only test the migration of the database tables.